lib/puppet/defaults.rb

```diff
@@ -1545,8 +1545,8 @@ def self.initialize_default_settings!(settings)
     :server_list => {
       :default => [],
       :type => :server_list,
-      :desc => "The list of puppet master servers to which the puppet agent should connect,
-        in the order that they will be tried.",
+      :desc => "The list of Puppet master servers to which the Puppet agent should connect,
+        in the order that they will be tried. Values must be comma-separated. Default is an empty string.",
```

Comment:
We should call out that each value should be a fully qualified domain name, which may be followed by an optional  ':' and port number. If a port is omitted, then puppet will use `masterport` for that host. So the following examples are valid:

```
puppet1.example.com,puppet2.example.com
puppet1.example.com:8141,puppet2.example.com:8140
puppet1.example.com:8141,puppet2.example.com
```